gobblin-hive-registration/src/test/java/gobblin/hive/policy/HiveRegistrationPolicyBaseTest.java

```diff
@@ -63,6 +63,40 @@ public void testGetHiveSpecs() throws IOException {
     examine(spec, "db2", "tbl3");
   }
 
+  @Test
+  public void testGetHiveSpecsWithDBFilter() throws IOException{
```

Comment:
As discussed offline, since existing test covers the token it should be sufficient. Further added bit more java doc around the config declaration to make it clearer for developer readability